src/status_im/common/bottom_sheet/view.cljs

```diff
@@ -106,7 +106,11 @@
                     {:transform [{:translateY translate-y}]}
                     (style/sheet insets window-height selected-item))}
            (when shell?
-             [blur/ios-view {:style style/shell-bg}])
+             [blur/ios-view
+              {:style         style/shell-bg
+               :blur-radius   blur-radius
+               :blur-type     :transparent
+               :overlay-color :transparent}])
```

Comment:
I'm afraid this would affect the sheets which use `shell?` (most sheets in onboarding do use the `shell` theme).

Reply:
I think I'll add a new prop for this blurring only.

Reply:
Done